src/shogun/CMakeLists.txt

```diff
@@ -217,6 +217,21 @@ CHECK_FUNCTION_EXISTS(lgammal HAVE_LGAMMAL)
 
 ############################ external dependencies
 
+#Stan math
+include(external/StanMath)
+SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC SYSTEM
+  $<BUILD_INTERFACE:${STAN_INCLUDE_DIR_STAN_MATH}>
+  $<INSTALL_INTERFACE:include/shogun/lib/external/Stan>
```

Comment:
who and what will install those the folder here?

Reply:
i would actually start a new dir: `include/shogun/third_party` and start putting stuff there that are external... we have too many things in `lib`